opendebates/tasks.py

```diff
@@ -42,6 +72,24 @@ def update_recent_events():
 
         logger.debug("There are %d entries" % len(entries))
         logger.debug("There are %d votes" % number_of_votes)
+    except:
+        # This task runs too frequently to report a recurring problem every time
+        # it happens.
+        global exception_logged
```

Comment:
Yes, the idea is to only log the exception once per execution of a worker, so we don't get a flood of logged errors every time the task is run again.  Whether that's a good idea...?